src/test/java/api/requests/RequestsAPICreateMultipleRequestsTests.java

```diff
@@ -76,6 +84,41 @@ public void canCreateMultipleRequestsOfDifferentTypeForSameItem()
     assertThat(thirdRequest.getJson().getInteger("position"), is(3));
   }
 
+  @Test
+  public void cannotCreateMultipleRequestsWithPageRequestForSameItemWhenItIsCheckedOut()
```

Comment:
Is this test intending to demonstrate that we disallow page requests when an item is `Checked out`.

If it is, then I think that is better demonstrated by the specific test you have already added, as I'm not sure what the presence of other requests is intended to demonstrate. Can you help me understand what behaviour their presence is demonstrating?

Reply:
Here I'm trying to show the scenario of multiple requests can be created for a checked out item except for a Page request.  It's just another way of looking at the request rules.